docs/upgrade/v1-3-2-to-v1-4-0.md

```diff
@@ -112,5 +112,20 @@ You can run the following command to fix the issue.
 ```shell
 helm rollback fleet -n cattle-fleet-system <last-deployed-revision>
 ```
-
 ---
+
+### 4. Upgrade will start over again unexpectedly after clicking the "Dismiss it" button
+
+When you use Rancher to upgrade Harvester, the Rancher UI displays a dialog with a button labeled "Dismiss it". Clicking this button may trigger an unexpected restart of the upgrade process.
```

Comment:
>Clicking this button may trigger an unexpected restart of the upgrade process.

Clicking this button will wipe the `upgrade.harvesterhci.io` object's status, important information like below is lost.

```
  status:
   ...
    previousVersion: v1.5-cb719430-head
    repoInfo: |
      release:
        harvester: cb719430
        harvesterChart: 0.0.0-v1.5-cb719430
        os: Harvester v1.5
        kubernetes: v1.32.2+rke2r1
        rancher: v2.11.0-rc5
        monitoringChart: 105.1.2+up61.3.2
```
The another side-effect is: it may trigger the unexpected restart of the upgrade process when `upgradeLog` is not enabled. When `upgradeLog` is enabled, it will not.